src/algokit/core/sandbox.py

```diff
@@ -116,7 +116,16 @@ def ps(self, service_name: str | None = None) -> list[dict[str, Any]]:
         )
         if run_results.exit_code != 0:
             return []
-        data = json.loads(run_results.output)
+
+        data = []
```

Comment:
can this line be removed now? if that impacts the type checker, might just need to explicitly annotate the assignment in the if branch

Reply:
Removed in 9709b8564bc1f4bcc0f958e58fba452745f9ff23